pkg/cache/cache.go

```diff
@@ -40,6 +41,8 @@ type CacheResponse struct {
 type ServiceAccountCache interface {
 	Start()
 	Get(name, namespace string) (role, aud string, useRegionalSTS bool)
+	// List returns cache contents as JSON string
+	List() string
```

Comment:
I would call this `ToJSON` or `MarshalJSON` or something if we systematically expect json. Otherwise `String() string` could also work. With `List()` I am expecting a slice, which is not what is returned.

Reply:
Makes sense. ToJSON seems to fit the return type.